examples/splinterdb_custom_ipv4_addr_sortcmp_example.c

```diff
@@ -90,6 +85,10 @@ int num_inet_addrs = (sizeof(inet_addr_info) / sizeof(*inet_addr_info));
 
 // clang-format on
 
+// Define min/max key values for user-defined IP4-addr keys
+#define IP4ADDR_MIN_KEY "0.0.0.0"
```

Comment:
I really, only, needed to re-use this one. (See line 355 below, to handle the case of iterate-for-all kv-pairs when an initial value is not specified.)

While at it, pulled out a `#define` for the sibling, too.